weblate/accounts/models.py

```diff
@@ -419,7 +419,7 @@ class Profile(models.Model):
     DASHBOARD_COMPONENT_LISTS = 6
 
     DASHBOARD_CHOICES = (
-        (DASHBOARD_WATCHED, _('Watched translations')),
+        (DASHBOARD_WATCHED, _('Watched components')),
```

Comment:
Actually it's "Translations to your languages from watched components", so neither of these is accurate.

Reply:
**Watched component string changes in your languages**
is not good
**New translations from watched components in your languages**
is somewhat better

Doesn't seem to upset anything length-wise

![image](https://user-images.githubusercontent.com/13802408/65392364-64e22d80-dd74-11e9-9675-b600f8568280.png)

Reply:
This is about *Dashboard* - what will be default tab shown there. The same string is used as label there.